net.jenyay.Outwiker.yml

```diff
@@ -0,0 +1,36 @@
+app-id: net.jenyay.Outwiker
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: outwiker
+rename-icon: net.jenyay.Outwiker
+finish-args:
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
+modules:
+  - name: outwiker
+    buildsystem: simple
+    no-autogen: true
+    sources:
+      - type: git
+        url: https://github.com/Jenyay/outwiker
+        commit: 92735a55330df4350f02d8d60075ebb9e3efc47b
+        tag: flatpak-2
+    build-options:
+      build-args:
+        - --share=network
+    build-commands:
+      - "pip3 install --prefix=${FLATPAK_DEST} -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-18.04 wxPython==4.1.1"
+      - "pip3 install --prefix=${FLATPAK_DEST} Pillow==8.1.2 cyhunspell==2.0.2 idna>=2.10 pyparsing==2.4.7 psutil==5.8.0"
+      - make -f Makefile.flatpak install
```

Comment:
none of this will work, builders don't have access to the network.

There are other packages that do install wxWidget.

Reply:
How can I install the required packages?

Reply:
Using this:

https://github.com/flatpak/flatpak-builder-tools/tree/master/pip

Reply:
Thank you. It remains to solve the problem with the Cyhunspell library (https://pypi.org/project/cyhunspell/). flatpak-builder-tools cannot find the sources of this library on PyPi.

Reply:
sometime a bit of manual work is needed....

Reply:
in that case upstream doesn't have a source tarball on pypi.

Reply:
Filed an issue about it:
https://github.com/MSeal/cython_hunspell/issues/31